htdocs/product/inventory/inventory.php

```diff
@@ -603,13 +617,13 @@ function init_myfunc()
 				print '</td>';
 			}
 
-			print '<td class="right">';
+			print '<td class="right expectedqty" id="id_'.$obj->rowid.'">';
 			print $obj->qty_stock;
 			print '</td>';
 			print '<td class="center">';
 			if ($object->status == $object::STATUS_VALIDATED) {
 				$qty_view = GETPOST("id_".$obj->rowid) ? GETPOST("id_".$obj->rowid) : $obj->qty_view;
-				print '<input type="text" class="maxwidth75 right" name="id_'.$obj->rowid.'" value="'.$qty_view.'">';
+				print '<input type="text" class="maxwidth75 right" id="id_'.$obj->rowid.'_input" value="'.$qty_view.'">';
```

Comment:
You need to add the id for autofill but are you sure you can remove the "name" parameter ?
What happen if you then post the form ?